eol_contact_form/locale/en/LC_MESSAGES/djangojs.po

```diff
@@ -0,0 +1,164 @@
+# English translations for OPEN.
+# Copyright (C) 2024 EOL
+# This file is distributed under the same license as the OPEN project.
+# EOL <eol-ing@uchile.cl>, 2024.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: 0.1.0\n"
+"Report-Msgid-Bugs-To: eol-ing@uchile.cl\n"
+"POT-Creation-Date: 2024-08-29 17:29+0000\n"
+"PO-Revision-Date: 2024-08-29 17:29+0000\n"
+"Last-Translator: EOL <eol-ing@uchile.cl>\n"
+"Language: en\n"
+"Language-Team: en <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.6.0\n"
+
+#: eol_contact_form/tests.py:65 eol_contact_form/views.py:113
+msgid "Message"
+msgstr ""
+
+#: eol_contact_form/tests.py:70 eol_contact_form/views.py:108
+msgid "Course"
+msgstr ""
+
+#: eol_contact_form/tests.py:75 eol_contact_form/views.py:103
+msgid "Category"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:50
+#: eol_contact_form/tests.py:80 eol_contact_form/views.py:98
+msgid "Email"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:46
+#: eol_contact_form/tests.py:85 eol_contact_form/views.py:93
+msgid "Name"
+msgstr ""
+
+#: eol_contact_form/templates/emails/help_desk_email.txt:8
+msgid ""
+"(Username: {user_username}){strong_tag_close} has created a ticket "
+"through the platform:"
+msgstr ""
+
+#: eol_contact_form/templates/emails/help_desk_email.txt:10
+msgid "{user_name}{strong_tag_close} has created a ticket through the platform:"
+msgstr ""
+
+#: eol_contact_form/templates/emails/help_desk_email.txt:12
+msgid "User Email:"
+msgstr ""
+
+#: eol_contact_form/templates/emails/help_desk_email.txt:13
+msgid "Category:"
+msgstr ""
+
+#: eol_contact_form/templates/emails/help_desk_email.txt:15
+msgid "Previous Page:"
+msgstr ""
+
+#: eol_contact_form/templates/emails/help_desk_email.txt:16
+msgid "Previous Page Course Code:"
+msgstr ""
+
+#: eol_contact_form/templates/emails/help_desk_email.txt:19
+msgid "Course:"
+msgstr ""
+
+#: eol_contact_form/templates/emails/help_desk_email.txt:22
+msgid "Message body:"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:7
+#: eol_contact_form/templates/eol_contact_form/contact.html:18
+msgid "Contact Form"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:19
+msgid "Fill out the form and our help desk will contact you shortly."
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:22
+msgid "Help desk hours (GMT-4): from 09:00 until 18:00 (Monday - Friday)"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:27
+msgid "Try Again"
+msgstr ""
+
+#: eol_contact_form/templates/eol_contact_form/contact.html:27
+msgid "There was a problem trying to generate your ticket."
```

Comment:
"There was a problem trying to send your message". Cambiar formulario acorde. El error no es al generar el ticket, eso es otro sistema (OTRS) del cual el edx ni se entera. El problema ocurriría en este caso si es que el formulario no se puede enviar.